src/main/java/net/pms/network/SearchRequestHandler.java

```diff
@@ -82,7 +83,7 @@ public StringBuilder createSearchResponse(SearchRequest requestMessage, Renderer
 
 		int totalMatches = getDLNAResourceCountFromSQL(convertToCountSql(requestMessage.getSearchCriteria(), requestType));
 
-		VirtualFolderDbId folder = new VirtualFolderDbId(requestType, "Search Result", "");
+		VirtualFolderDbId folder = new VirtualFolderDbId("Search Result", new DbidTypeAndIdent(requestType, ""), "");
```

Comment:
Can "Search Result" be translatable?

Reply:
It can be translated, but this concrete virtual folder's name is not being delivered to the client. Strictly speaking, it can stay empty. I named the virtual folder "Search Result" just for debugging purposes ...